From 6d2c4ec121894ad7fe7c68ea71b705d7d22cba50 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Tue, 11 Aug 2026 09:23:51 +0000
Subject: [PATCH] core: Remove old app target SDK dialog

If an app is old, users should already know that, and there's usually no
point in warning them about it because they would already be using a
newer version if one existed. Sometimes, using an old app is necessary
for one reason or another, so remove this annoyance and let the user use
their old app in peace.

Change-Id: I1a3021f0f9bec1ab6ff7641776391e1bd4c3cc49

[Port to A16] Single line removal; context drifted (16kb page-size flag added upstream).

Change-Id: I7bf97acde2dc9c3ad1c7f07ee21b3216a1eb8ed9
---
 services/core/java/com/android/server/wm/AppWarnings.java | 1 -
 1 file changed, 1 deletion(-)

diff --git a/services/core/java/com/android/server/wm/AppWarnings.java b/services/core/java/com/android/server/wm/AppWarnings.java
index 995e3a9feb7e..e575af289c2d 100644
--- a/services/core/java/com/android/server/wm/AppWarnings.java
+++ b/services/core/java/com/android/server/wm/AppWarnings.java
@@ -273,7 +273,6 @@ class AppWarnings {
     public void onStartActivity(ActivityRecord r) {
         showUnsupportedCompileSdkDialogIfNeeded(r);
         showUnsupportedDisplaySizeDialogIfNeeded(r);
-        showDeprecatedTargetDialogIfNeeded(r);
         showDeprecatedAbiDialogIfNeeded(r);
         if (Flags.appCompatOption16kb()) {
             showPageSizeMismatchDialogIfNeeded(r);
-- 
2.43.0

